docker/django/Dockerfile

```
@@ -0,0 +1,38 @@
+# ==============================
+FROM python:3.7 as base
+# ==============================
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN useradd -Ums /bin/bash -d /appuser appuser
+
+WORKDIR /app
+
+COPY --chown=appuser:appuser ../../requirements.txt .
+
+# Allows installation of postgresql-client-10 on Debian stretch which the Python images are based on
+RUN echo 'deb https://apt-archive.postgresql.org/pub/repos/apt/ stretch-pgdg main' > /etc/apt/sources.list.d/pgdg.list
+RUN wget -q -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+RUN apt-get update && apt-get install -y \
+    gettext \
+    postgresql-client-10 \
+    && pip install -U pip \
+    && pip install --no-cache-dir -r requirements.txt
+
+# ==============================
+FROM base as development
+# ==============================
+
+COPY --chown=appuser:appuser ../../dev-requirements.txt .
+
+RUN pip install -r dev-requirements.txt --no-cache-dir
+
+COPY --chown=appuser:appuser ../.. /app/
```

Could this even be? :thinking: 
```suggestion
COPY --chown=appuser:appuser . .
```